### PR TITLE
DOCSP-4898 - v2.10.0 release notes.

### PR DESCRIPTION
**Pull request by jdestefano-mongo**
JIRA: 

Staging: https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/jdestefano/DOCSP-4898/release-notes.html#bi-2-10-0

Related docs-tools PR: https://github.com/mongodb/docs-tools/pull/271

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -20,6 +20,104 @@ Release Notes for |bi|
...
+Bug Fixes
+~~~~~~~~~
+
+- Fixes a bug where aggregate functions returned ``NULL`` in dual
+  queries with the ``DUAL`` keyword omitted.
+
+- Fixes a bug where some invalid ``DUAL`` queries did not return an
+  error.
+
+- Fix of a bug that prevented pushdown of some joins to derived tables
```
Comment (rychipman) [inline source/release-notes.txt]: for consistency, should this also be "fixes" instead of "fix of"?

File: source/release-notes.txt [hunk 2]
```diff
@@ -20,6 +20,104 @@ Release Notes for |bi|
...
+  4.0.
+
+Date and Time Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+This release resolves a number of inconsistencies in the |bi-short|'s
+date and timestamp semantics. Resolving these inconsistencies resulted
+in some changes to user-facing behavior:
+
+- Removed inconsistent support for partial dates.
```
Comment (rychipman) [inline source/release-notes.txt]: The tense here doesn't match the previous section -- is that intentional?

File: source/release-notes.txt (hunk 1 above)
Comment (jdestefano-mongo) [reply source/release-notes.txt]: Good catch

File: source/release-notes.txt (hunk 2 above)
Comment (jdestefano-mongo) [reply source/release-notes.txt]: Good catch

**Pull request merged**